src/aat/java/uk/gov/hmcts/reform/em/annotation/functional/AnnotationScenarios.java

```diff
@@ -4,30 +4,45 @@
 import org.json.JSONArray;
 import org.json.JSONObject;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.PropertySource;
 import org.springframework.http.MediaType;
-import uk.gov.hmcts.reform.em.annotation.testutil.Env;
+import org.springframework.test.context.junit4.SpringRunner;
+import uk.gov.hmcts.reform.em.EmTestConfig;
 import uk.gov.hmcts.reform.em.annotation.testutil.TestUtil;
 
-import static org.hamcrest.CoreMatchers.*;
 import java.util.UUID;
 
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.is;
+
+@SpringBootTest(classes = {TestUtil.class, EmTestConfig.class})
+@PropertySource(value = "classpath:application.yml")
+@RunWith(SpringRunner.class)
```

Comment:
I really tried hard not to boot spring for these tests, but I guess you can fight it for ever

Reply:
Indeed. All the reform client libs (idam, dm, ccd) use spring context